src/main/io/osd.c

```diff
@@ -293,7 +293,7 @@ static void osdDrawSingleElement(uint8_t item)
             elemPosX = 14;
             elemPosY = 6 - 4; // Top center of the AH area
 
-            int rollAngle = attitude.values.roll;
+            int rollAngle = -attitude.values.roll;
```

Comment:
Good spot. Yes, AHI should be reversed compared to aircraft roll. Same is probably true for pitch, but that depends on how the line is drawed.